dff/script/extra_types.py

```diff
@@ -0,0 +1,15 @@
+from typing_extensions import Unpack
+from pydantic import ConfigDict
+from dff.script.core.message import Attachment
+
+
+class Poll(Attachment):
```

Comment:
@pseusys 
We've discussed an improvement to the first approach:
Messenger Interfaces can create child classes for attachments that define all available options.
Then a Messenger Interface would convert a regular instance to the child instance with extra fields becoming regular ones.
That way we can pass a `Poll` with specified background image in a script used by vk and tg interfaces and tg interface would ignore the background image parameter while vk interface would send it to the API.
If user has no need of adding multiple messenger interface support to their script they can just use `VKPoll` instead of a regular one.